*
* Visual Studio Code Server
*
* By using the software, you agree to
* the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
* the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
*
Server bound to 127.0.0.1:40563 (IPv4)
Extension host agent listening on 40563

[07:14:07] 




[07:14:07] Extension host agent started.
[07:14:07] [127.0.0.1][1492b1a0][ManagementConnection] New connection established.
[07:14:07] [127.0.0.1][28d144d8][ExtensionHostConnection] New connection established.
[07:14:07] [127.0.0.1][28d144d8][ExtensionHostConnection] <119131> Launched Extension Host Process.
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[07:14:10] Error [CodeExpectedError]: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateIcon (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27562)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[07:14:11] Error [CodeExpectedError]: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateIcon (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27562)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[07:14:11] Error [CodeExpectedError]: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[07:14:12] Error [CodeExpectedError]: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[07:14:23] Error [CodeExpectedError]: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
New EH opened, aborting shutdown
[07:19:07] New EH opened, aborting shutdown
[08:58:06] [127.0.0.1][1492b1a0][ManagementConnection] The client has reconnected.
[08:58:06] [127.0.0.1][28d144d8][ExtensionHostConnection] The client has reconnected.
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateIcon (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27562)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[08:58:07] Error [CodeExpectedError]: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateIcon (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27562)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[08:58:07] Error [CodeExpectedError]: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[08:58:07] Error [CodeExpectedError]: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[08:58:08] Error [CodeExpectedError]: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[08:58:28] [127.0.0.1][1492b1a0][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[08:58:28] [127.0.0.1][28d144d8][ExtensionHostConnection] <119131> Extension Host Process exited with code: 0, signal: null.
Last EH closed, waiting before shutting down
[08:58:28] Last EH closed, waiting before shutting down
Got delay-shutdown request while in shutdown timeout, delaying
[08:58:39] Got delay-shutdown request while in shutdown timeout, delaying
Cancelling previous shutdown timeout
[08:58:39] Cancelling previous shutdown timeout
[08:58:39] [127.0.0.1][303d4b72][ManagementConnection] New connection established.
[08:58:39] [127.0.0.1][670f07c5][ExtensionHostConnection] New connection established.
[08:58:39] [127.0.0.1][670f07c5][ExtensionHostConnection] <139731> Launched Extension Host Process.
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateIcon (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27562)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[08:58:44] Error [CodeExpectedError]: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateIcon (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27562)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[08:58:44] Error [CodeExpectedError]: Could not find pty on pty host
    at A.U (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:32377)
    at A.updateTitle (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:27509)
    at T.<computed> (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:43:22735)
    at Object.call (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at g.s (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at g.q (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at i.y (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at i.fire (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ce (/home/toto/.vscode-server/bin/af28b32d7e553898b2a91af498b1fb666fdebe0c/out/vs/platform/terminal/node/ptyHostMain.js:25:10117)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
New EH opened, aborting shutdown
[09:03:39] New EH opened, aborting shutdown
